src/Psr17/MessageFactory.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Http\Message\Psr17;
+
+use Psr\Http\Message\RequestFactoryInterface;
+use Psr\Http\Message\ResponseFactoryInterface;
+use Psr\Http\Message\StreamFactoryInterface;
+
+/**
+ * Compatibility layer for PSR-17 reqeust and response factory.
+ *
+ * @author Márk Sági-Kazár <mark.sagikazar@gmail.com>
+ */
+final class MessageFactory implements \Http\Message\MessageFactory
```

Comment:
i usually `use` such a thing and specify an alias like BaseMessageFactory, or in this case maybe LegacyMessageFactory?